testbed/testbed/load_generator.go

```diff
@@ -162,7 +173,7 @@ func (lg *LoadGenerator) generate() {
 
 		go func() {
 			defer workers.Done()
-			t := time.NewTicker(time.Second / time.Duration(lg.options.DataItemsPerSecond/lg.options.ItemsPerBatch/numWorkers))
+			t := time.NewTicker(lg.options.Interval / time.Duration(lg.options.DataItemsPerInterval/lg.options.ItemsPerBatch/numWorkers))
```

Comment:
It is not clear why introduce options.Interval if that is not the actual interval between generations. The naming of options Interval/DataItemsPerInterval does not seem to accurately describe the behavior.
I think we can keep DataItemsPerSecond as is, and delete Interval which is not used anywhere else but this particular calculation. I don't see why the changes to options is necessary for this PR.

Reply:
Thanks @tigrannajaryan, I made this change to match the scrape interval chosen which Prometheus uses. I'll change it back to keep DataItemsPerSecond as is here and instead calculate DataItemsPerSecond based off the scrape interval in the Prometheus test

Reply:
@tigrannajaryan I have now pushed this change